README.md

```diff
@@ -138,6 +138,8 @@ After installing, you can run `darwin-rebuild` to apply changes to your system:
 darwin-rebuild switch --flake ~/.config/nix-darwin
 ```
 
+NOTE: if the 'darwin-rebuild' command is not found, you will need to add a path to your ~/.bashrc or ~/.zshrc file. In the file, add: `export PATH=/run/current-system/sw/bin:$PATH`
+
```

Comment:
This line should replace the one on 135

```suggestion
After installing, restarting your computer should make `darwin-rebuild` available in your `PATH` to apply changes to your system:
```

Reply:
@Enzime I've incorporated your suggestion